_posts/2020-04-28-US-RSE-progress.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: A year of progress for US-RSE
+tags: [communication]
+---
+
+by Daniel S. Katz, Vanessa Sochat, Charles Ferenbaugh, Jeffrey C. Carver, Sandra Gesing, Ian Cosden, Christina Maimone, and Chris Hill
+
+The US-RSE association was first planned in 2018, but it really became active and started growing in 2019. In the last year or so, we've made a lot of progress and have a number of activities that are in-progress. In this post, we want to report on this progress.
+
+## Accomplishments
+
+**Membership**. In January 2019, we had 10 members, who were all involved in starting up US-RSE. By the end of 2019, this had grown to 259, and is now (at the end of March 2020) 345.
+
+<p align=center>![US-RSE Membership]({{ site.baseurl }}/assets/posts/2020-04-28-us-rse-membership.png "US-RSE Membership")<br/>_US-RSE Membership_</p>
```

Comment:
Since this it html + markdown it's going to render funny, and it does:

![image](https://user-images.githubusercontent.com/814322/80535757-8cc6ee80-895e-11ea-9b05-330cf7180230.png)

Instead try using a div with style (and tweak this as you need). You can also add style elements directly to the img, if that's appropriate.

```html
<div style="align:center">
    <img src="{{ site.baseurl }}/assets/posts/2020-04-28-us-rse-membership.png" alt="US-RSE Membership">
</div>
```

Reply:
The image was fine originally - what I was trying to do was to add the caption immediately under the image, ideally centered and italic.  Do you have a suggestion of how to do this?

Reply:
let me try one thing...

Reply:
I've added captions now